dotCMS/src/main/java/com/dotcms/analytics/metrics/MetricType.java

```diff
@@ -29,9 +29,14 @@ public enum MetricType {
                     Parameter.builder().name("target").build()
             )
             .availableOperators(Operator.EQUALS, Operator.CONTAINS)),
+    EXIT_RATE(new Builder()
+            .label("Exit Rate")
+            .optionalParameters(Parameter.builder().name("url").build())),
+
     BOUNCE_RATE(new Builder()
```

Comment:
Shouldn't this be removed?

Reply:
Really yes, but really I add it again on the next PR so ...